ironmon_tracker/screens/LogOverlay.lua

```diff
@@ -408,7 +412,39 @@ LogOverlay.NavFilters = {
 }
 
 function LogOverlay.initialize()
+
 	LogOverlay.TabHistory = {}
+
+	local preEvoSettingButton = {
+		type = Constants.ButtonTypes.CHECKBOX,
+		text =  LogOverlay.preEvoSetting,
+		textColor = "Default text",
+		boxColors = { "Lower box border", "Lower box background" },
+		tab = LogOverlay.Tabs.MISC,
+		box = {
+			LogOverlay.margin + 3,
+			110,
+			Constants.Font.SIZE-1,
+			Constants.Font.SIZE-1,
+		},
+		clickableArea = {
+			LogOverlay.margin + 3,
+			110,
+			Utils.calcWordPixelLength(LogOverlay.preEvoSetting) + 10,
+			Constants.Font.SIZE,
+		},
+		isVisible = function(self)
+			return LogOverlay.currentTab == self.tab
+		end,
+		toggleState = Options[LogOverlay.preEvoSetting],
+		toggleColor = "Positive text",
+		onClick = function(self)
+			self.toggleState = not self.toggleState
+			Options.updateSetting(self.text, self.toggleState)
+		end,
+	}
+
+	table.insert(LogOverlay.Buttons, preEvoSettingButton)
```

Comment:
You should create this button with all the other buttons outside of the initialize() function. After doing this, add a line of code inside this initialize() function to apply the loaded setting to update the button and its state.

`TimeMachineScreen.Buttons.EnableRestorePoints` is an example of how this is done. Check the `TimeMachineScreen.initialize()` function to see how the setting gets applied after the Tracker finishes setting up and has all the settings loaded from the Settings.ini file.

Reply:
Moved per your suggestion in latest batch of commits